modules/theme-tailwind-magento-px/molecules/AddToCart.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <Button
+    color="primary"
+    :disabled="isLoading"
+    data-cy="add-to-cart"
+    class="relative w-full sm:w-auto sm:h-auto mt-4 sm:mt-0 sm:ml-3 text-bold"
+    :class="product.available || 'opacity-50 pointer-events-none'"
+    :data-simple-product="product.isSimpleProduct && !product.isOptionsProduct"
+    @click="onAddToCart"
+  >
+    <slot>
+      <span v-if="!isLoading">{{ t(isEnabled ? 'Add' : 'Conf') }}</span>
+      <Spinner v-if="isLoading" />
+    </slot>
+
+    <CrossSellModal v-if="isCrossSellModalOpen" @close="onClose" />
+  </Button>
+</template>
+
+<script setup lang="ts">
```

Comment:
Do not duplicate components just because they use some event emitter. There must be a better way.